pkgs/applications/audio/jamesdsp/0002-src.pro-added-PREFIX-config-var.patch

```diff
@@ -0,0 +1,55 @@
+From 003c9e9fc426f83e269aed6e05be3ed55273931a Mon Sep 17 00:00:00 2001
```

Comment:
You can now use `fetchpatch` with `https://github.com/Audio4Linux/JDSP4Linux/commit/003c9e9fc426f83e269aed6e05be3ed55273931a.patch`